### PR TITLE
Server create message

### PR DESCRIPTION
**Pull request by stephen-lt-sl**
@stevenhillcox @sievins @jebbinSCL

Adds endpoint to the server that allows for messages to be created

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -90,5 +90,45 @@ module.exports = function(port, db, githubAuthoriser) {
...
 
+    app.post("/api/messages", function(req, res) {
+        var senderID = req.session.user;
+        var conversationID = req.body.conversationID;
+        var message = req.body.contents;
+        if (message === "") {
+            res.sendStatus(201);
+            return;
+        }
+        conversations.findOne({
```
Comment (jebbinSCL) [inline server/server.js:158]: Fine for now, but might be worth considering how to use promises instead of callbacks here.

**Comment by jebbinSCL** [comment]
:ship:  after considering comment.

**Comment by stephen-lt-sl** [comment]
Just rebased to start off the current master, no actual changes made since your last comment

**Comment by jebbinSCL** [comment]
Are you making that change from the comment? Otherwise its fine to merge.

**Comment by stephen-lt-sl** [comment]
I am planning to change the server logic to use promises rather than callbacks - but I think it'd be preferable to do so after all the feature branches have merged to master, as converting all the db operations to promises is likely a branch-worthy task in itself

**Pull request merged**